### PR TITLE
Data for automate sign up to serve

### PR DESCRIPTION
**Pull request by candrews-thrive**
Added two new groups with two opportunities
Add a new user and set the user as a participant for the two groups

### CODE REVIEW

File: CI/TestSQL/02.TestData/AvalancheSU2S.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+USE [MinistryPlatform]
+GO
+
+INSERT INTO Groups 
+(Group_Name                                    ,Group_Type_ID,Ministry_ID ,Congregation_ID,Primary_Contact,Description,Start_Date                ,Target_Size,Group_Is_Full,Available_Online,Domain_ID,[Secure_Check-in],Suppress_Nametag,Suppress_Care_Note,On_Classroom_Manager,Promote_Weekly,__IsPublic,__ISBlogEnabled,__ISWebEnabled,Group_Notes,Sign_Up_To_Serve,Deadline_Passed_Message_ID,Send_Attendance_Notification,Send_Service_Notification,Child_Care_Available,Kids_Welcome) VALUES
+('(t+auto) Avalanche',9            ,11          ,1              ,7654100        ,'Test data',{ts '2015-11-21 00:00:00'},0          ,0            ,1               ,1        ,0                ,0               ,0                 ,0                   ,0             ,'N'       ,'Y'            ,'Y'           ,''         ,null            ,null                      ,0                           ,0                        ,0                   ,0           );
```
Comment (lippsm) [inline CI/TestSQL/02.TestData/AvalancheSU2S.sql]: Adjust spacing to line up the commas to make this more readable. Same goes for all statements like this.

File: CI/TestSQL/02.TestData/AvalancheSU2S.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+--Saturday Attack Shinra Mako reactor 1
+--Setup
+INSERT INTO Opportunities 
+(Opportunity_Title                     ,Description,Group_Role_ID,Program_ID,Visibility_Level_ID,Contact_Person,Publish_Date              ,Add_to_Group                                                          ,Minimum_Needed,Maximum_Needed,[Domain_ID],On_Connection_Card,Shift_Start   ,Shift_End     ,Event_Type_ID,Sign_Up_Deadline_ID,Room           ,Send_Reminder,Reminder_Template,Reminder_Days_Prior) VALUES
+('(t+auto) Attack Shinra Mako reactor 1','Test Data',16           ,106       ,4                  ,7654100       ,{ts '2015-11-27 14:55:00'},(select group_id from Groups where Group_Name = '(t+auto) Avalanche'),1             ,250           ,1          ,0                 ,{t '14:00:00'},{t '16:00:00'},94           ,7                  ,'Oakley Atrium',1            ,108700           ,3                  );
```
Comment (lippsm) [inline CI/TestSQL/02.TestData/AvalancheSU2S.sql]: Consider using variables instead of subselects. i.e Add_To_Group field. Same goes for all subselects.

**Comment by candrews-thrive** [comment]
@lippsm  I have fixed all of the issues in your comment.

**Comment by hookom** [comment]
I'm good with it when @lippsm is.

**Comment by lippsm** [comment]
Merged

**Pull request merged**